### DISCUSSION TITLE
[Enhancement] [select] tabindex to ElSelect

### DISCUSSION THREAD

**Discussion by SparK-Cruz** [element-plus/element-plus#19030] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

ElSelect gets skipped in form because if you inform a tabindex it will focus on the wrapper element and not the inner editable one that gets focus when no tabindex is there, but then if the form uses tabindex the element is picked last.

### Features
1. When a tabindex attribute is present allow it to focus the right part of the element that makes it editable with arrow keys.

### Additional comments

_(empty)_

**Comment by tolking** [comment]
This is a BUG and will be fixed in the next version v2.9.0

Link #19034
